gcsio/src/main/java/com/google/cloud/hadoop/gcsio/testing/FakeServer.java

```diff
@@ -0,0 +1,45 @@
+package com.google.cloud.hadoop.gcsio.testing;
```

Comment:
Please add license header. Also for other files.

Reply:
Done